example/example.spec.js

```diff
@@ -0,0 +1,77 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+const path = require('path');
+
+const getTextContent = async (element) => {
+	return await element.evaluate((el) => el.textContent);
+};
```

Comment:
the default `toHaveText` does not work as expected with slots that have default content. There is probably an issue to raise in the playwright repository, but for the moment, just using this custom function to evaluate the results of `textContent` manually.